src/Explorer/Tree/ResourceTreeAdapter.tsx

```diff
@@ -245,6 +246,11 @@ export class ResourceTreeAdapter implements ReactAdapter {
       isSelected: () => this.isDataNodeSelected(collection.rid, "Collection", ViewModels.CollectionTabKind.Settings)
     });
 
+    let schemaNode: TreeNode = this.buildSchemaNode(collection);
```

Comment:
`const schemaNode`. And with `const` we usually drop the type and let the compiler infer unless you want to be verbose.